hi-ml/src/health_ml/utils/common_utils.py

```diff
@@ -256,10 +256,10 @@ def check_conda_environments(env_files: List[Path]) -> None:
 
     :param env_files: The list of Conda environment YAML files to check.
     """
-    if is_himl_used_from_git_repo():
-        repo_root_yaml: Optional[Path] = git_repo_root_folder() / ENVIRONMENT_YAML_FILE_NAME
+    if paths.is_himl_used_from_git_repo():
+        repo_root_yaml: Path = paths.shared_himl_conda_env_file()
     else:
-        repo_root_yaml = None
+        repo_root_yaml = None  # type: ignore
```

Comment:
the type:ignore should not be necessary if you undo the typing change in line 260?